### ISSUE TITLE
Possible Memory Leak

### ISSUE THREAD

**Issue by eugenkiss** [eugenkiss/chanobol#33]
There might be a memory leak. Because when revisiting the posts view after a couple of hours when it was in the background scrolling is stuttering much more. If you dismiss the posts view to get back to the catalog view, the catalog view also stutters when scrolling. But this could also be down to the fact that as many threads are dead almost all the card views are half-transparent which could reduce drawing performance. After refreshing the catalog view everythings plenty fast again.

**Comment by eugenkiss** [comment]
So testing if the transparent card views are the culprit or not is easy. But then again, why would the transparency of the cardviews of the catalog influence the performance of the thread view (postsview). It's on top of the catalog resp. it covers the catalog after all so Android shouldn't invest rendering performance into the catalog view...

**Comment by eugenkiss** [comment]
So I just set all thread cardviews on catalog view to transparent and sure enough scrolling is jittery. Yet clicking on a thread cardview which opens the posts view and scrolling there is smooth as butter. So there might be some kind of memory leak after all. On the other hand, I've recently replaced picasso, retrofit & okhttp with ion so if some of these were responsible it shouldn't happen anymore. In any case, need to watch out if it happens again.
